config.yml

```diff
@@ -26,6 +26,27 @@ layoutDir:              "site/layouts"
 staticDir:              "site/static"
 themesDir:              "site/themes"
```

Comment:
OK, I have checked the Hugo code. I have tried to be really careful about not breaking existing sites, so the logic is this:

* If there is a legacy config setting for a given target (e.g. `static`), we're ignoring any mounts.

So, in your case, you need to remove the `staticDir` setting above (also if you have others).

Reply:
I still can't make it to work, so I'll put it on hold until you are back and we figure this out.

If we manage to skip the copying step and also generate the SRI hashes natively in Hugo, this would be a huge win for us.

BTW, Thank you for all the hard work :)

Reply:
I will put this issue in the back of my head, but please ping me next week if I forget. What you want to do should not be a problem, and I don't think there should be any bug to prevent you from doing it, either, but one never knows ...

Reply:
OK, I found a related bug in Hugo. @XhmikosR could I get commit access to this PR so I can use this to test that fix? You will still have to wait for Hugo 0.56.3, but that should be a day or two.

Reply:
Unfortunately this branch is on the main repo so I can't give you rights. Can you fork the main repo and push to this branch on your fork?

Reply:
Sure.